spectacles/lookml.py

```diff
@@ -555,4 +592,6 @@ async def build_project(
     else:
         project = Project(name, [m for m in models if len(m.explores) > 0])
 
+    # Indicates whether the project has all of the models/explores or just the selected ones
+    project.is_complete_project = is_complete_project
```

Comment:
There is notionally a world where you have manually passed in _every_ explore within a project and this is true, even though `get_full_project` was passed as false. This may not matter, but wanted to flag it.